src/renderers/webgl/WebGLTextures.js

```diff
@@ -1270,6 +1271,72 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 	}
 
+	function mapTextureMapping( texture, mapping ) {
+
+		if ( mapping === EquirectangularReflectionMapping ) {
+
+			texture.mapping = CubeReflectionMapping;
+
+		} else if ( mapping === EquirectangularRefractionMapping ) {
+
+			texture.mapping = CubeRefractionMapping;
+
+		}
+
+	}
+
+	function getCubeTexture( envMap ) {
+
+		if ( envMap && ( envMap.isCubeTexture || envMap.isWebGLCubeTexture ) ) {
+
+			return envMap;
+
+		} else if ( envMap && envMap.isTexture ) {
+
+			const mapping = envMap.mapping;
+			const isDeprecatedEquirectangular = mapping === EquirectangularReflectionMapping || mapping === EquirectangularRefractionMapping;
```

Comment:
Maybe adding warning when `isDeprecatedEquirectangular` is `true`?

Reply:
Yes!